doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -128,14 +128,54 @@ Monitor the log file in a new terminal with:
 
     tail -f convert-gaia.log
 
-Check the log ouput after several hours: ``ConvertReferenceCatalogTask`` reports progress in 1% intervals.
+Check the log output after several hours: ``ConvertReferenceCatalogTask`` reports progress in 1% intervals.
 
 .. _lsst.meas.algorithms-refcat-ingest:
 
 4. Ingest the files into the butler
 ===================================
 
-When ``convertReferenceCatalog`` has finished, it will print the two commands you need to run to register the new refcat dataset type and ingest your converted output into it.
+When ``convertReferenceCatalog`` has finished, a new directory (named ``gaia-refcat/`` in the example above) will now exist containing all reference catalogs in the LSST format.
+
+.. note::
+
+    If you have an already existing converted reference catalog on disk (for example, the PS1 or Gaia DR2 catalogs that were used in gen2 butlers), you will first need to create a suitable filename to htm7-index astropy lookup table before running the commands in the remainder of this section.
+
+    This is an example script that constructs conversion lookup tables for all converted FITS files in the `gaiadr2` and `panstarrsps1` directories, outputting to the `outdir` directory:
+
+    .. code-block:: python
+
+        import os
+        import glob
+        import astropy.table
+
+        refcat_dirs = [gaiadr2, panstarrsps1]
+
+        # loop over each FITS file in all refcats
+        # note: this constructs a series of .ecsv files, each containing two columns:
```

Comment:
It's not "a series of ecsv files", it's one per refcat.

Thinking on this: I think this script should have a `refcat="gaiadr2"` or whatever at the top, and only a single for loop below. This document is about converting/ingesting a single refcat, and we should keep it that way in this note.